.github/workflows/archive-live-smoke.yml

```
@@ -0,0 +1,39 @@
+permissions:
+  contents: read
+
+jobs:
+  archive-live-smoke:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    environment: archive-live-smoke
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)